README.md

```diff
@@ -6,11 +6,11 @@ This directory contains the manifest files needed to deploy a standard maintenan
 This directory also contains the maintenance page HTML file, along with a DockerFile to compile an image.
 
 ## Deploying the page
-Within the `kubectl_deploy` directory, there are 2 simple manifest files that make up the deployment, `deploy.yaml` and `service.yaml`.
```

Comment:
you’ll need to rename the folder to match this

Reply:
the folder is already named `kubect_deploy`, happy to update the folder to be `kubectl_deploy` and not update the readme?

Reply:
hahaha i'm blind! i somehow thought your rename was in the opposite direction. ignore me :)

Reply:
Thanks @carlov20  for the PR. I think it is better to update the folder to `kubectl_deploy` so it doesnot look like a typo.